src/set_relation/set_relation.cc

```diff
@@ -2619,11 +2619,13 @@ void Conjunction::acceptVisitor(Visitor *v) {
     
     std::list<Exp*>::iterator expIter = mEqualities.begin();
     while (expIter != mEqualities.end()) {
+        (*expIter)->setEquality();
```

Comment:
Why is this necessary?  Aren't the equalities being set when they are inserted into the Conjunction?  We shouldn't have to do it here as well.